module4/homework/appendNewRecipe/AppendNewRecipe.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <vector>
+#include <list>
+#include <deque>
+#include <string>
+#include <utility>
+#include <sstream>
+
+using vecOfStrings = std::vector<std::string>;
+using listOfStrings = std::list<std::string>;
+using dequeOfPairs = std::deque<std::pair<size_t, char>>;
```

Comment:
Why? These aliases don't change anything. You name std::vector<std::string> VecOfString which mean exactly the same.